pkg/bot/mattermost.go

```diff
@@ -91,11 +91,20 @@ func NewMattermost(log logrus.FieldLogger, cfg config.Mattermost, executorFactor
 	client := model.NewAPIv4Client(cfg.URL)
 	client.SetOAuthToken(cfg.Token)
 
-	botTeam, resp := client.GetTeamByName(cfg.Team, "")
+	botTeams, resp := client.SearchTeams(&model.TeamSearch{
+		Term: cfg.Team,
+	})
```

Comment:
What you're doing here makes sense. But, I have a quick question: shouldn't this just be a documentation update?

That is, can we just ask users to add the Team name, not display name, in `values.yaml` - or is it hard to obtain that the actual team name?

Reply:
The problematic part is, Mattermost does not expose `name` field to UI. Of course we can say user users to execute the curl request I provided in PR description, but it adds additional complexity to flow. I thought same as you did but it was my findings, Wdyt?

Reply:
Thx. You've answered my question. Works for me.